<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsWK6500Meter</class>
 <widget class="QDialog" name="clsWK6500Meter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(150,150,251);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">
font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>主参数</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>项目1：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnItem1">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>6666666</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>L</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>单位：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbItem1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>56</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>68</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>上下限:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="clsLabel" name="labelLimit1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
	
	
	background-color: rgb(203, 201, 239);
	border:1px solid rgb(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>115.66--118.88</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true">
font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>副参数</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>项目2：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnItem2">
          <property name="minimumSize">
           <size>
            <width>68</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>Q</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>单位：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbItem2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>56</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>68</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>上下限:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="clsLabel" name="labelLimit2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>43</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
background-color: rgb(203, 201, 239);
	border:1px solid rgb(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string>6.00--88</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>频率电平</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>频率：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFreq">
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>100.00kHz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>电平：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLevel">
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>1.00V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>档位速度</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btnRange">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSpeed">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>Fast</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>18</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnEqucct">
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>666666</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>串联</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxBias">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>偏置设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>类型：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBiasType">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>电压</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBias">
          <property name="minimumSize">
           <size>
            <width>108</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>1.00V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBiasSwitch">
          <property name="minimumSize">
           <size>
            <width>39</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>开</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxBias_2">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>Relay 设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnRelay">
          <property name="minimumSize">
           <size>
            <width>39</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>开</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>104</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>clsLabel</class>
   <extends>QLabel</extends>
   <header location="global">clsLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
